tst/gstreamer/gstkvstest.cpp

```diff
@@ -217,20 +292,47 @@ GST_START_TEST(check_playing_to_paused_and_back_to_playing)
         fail_unless_equals_int(gst_element_change_state(pElement, GST_STATE_CHANGE_PAUSED_TO_PLAYING), GST_STATE_CHANGE_SUCCESS);
         fail_unless_equals_int(gst_element_change_state(pElement, GST_STATE_CHANGE_PLAYING_TO_PAUSED), GST_STATE_CHANGE_SUCCESS);
         fail_unless_equals_int(gst_element_change_state(pElement, GST_STATE_CHANGE_PAUSED_TO_PLAYING), GST_STATE_CHANGE_SUCCESS);
+
+        gst_pad_set_active (srcpad, FALSE);
+
         cleanup_kinesisvideoproducersink(pElement);
     }
 GST_END_TEST;
 
+GST_START_TEST(test_check_credentials)
+    {
+        ck_abort_msg("Required environment variables (ACCESS_KEY and/or SECRET_KEY) are not set");
```

Comment:
can it be extended to catch specifics instead of AK OR SK?

Reply:
Yes